### ISSUE TITLE
Install ros-kinetic-fzi-icl-can after ros-kinetic-libpcan duplicates the libpcan.so.0 library

### ISSUE THREAD

**Issue by clalancette** [fzi-forschungszentrum-informatik/fzi_icl_can#1]
If I install ROS kinetic, and then do:

```
$ sudo apt-get install ros-kinetic-libpcan
$ sudo apt-get install ros-kinetic ros-kinetic-fzi-icl-can
```

I get the following error message:

```
The following NEW packages will be installed:
  ros-kinetic-fzi-icl-can
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/75.3 kB of archives.
After this operation, 357 kB of additional disk space will be used.
(Reading database ... 260158 files and directories currently installed.)
Preparing to unpack .../ros-kinetic-fzi-icl-can_1.0.10-0xenial-20170807-101512-0800_amd64.deb ...
Unpacking ros-kinetic-fzi-icl-can (1.0.10-0xenial-20170807-101512-0800) ...
dpkg: error processing archive /var/cache/apt/archives/ros-kinetic-fzi-icl-can_1.0.10-0xenial-20170807-101512-0800_amd64.deb (--unpack):
 trying to overwrite '/opt/ros/kinetic/lib/libpcan.so.0', which is also in package ros-kinetic-libpcan 0.6.11-0xenial-20170717-080450-0800
Processing triggers for libc-bin (2.23-0ubuntu9) ...
/sbin/ldconfig.real: /usr/lib/nvidia-375/libEGL.so.1 is not a symbolic link

/sbin/ldconfig.real: /usr/lib32/nvidia-375/libEGL.so.1 is not a symbolic link

Errors were encountered while processing:
 /var/cache/apt/archives/ros-kinetic-fzi-icl-can_1.0.10-0xenial-20170807-101512-0800_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

It would probably be best not to vendor the libpcan library at all, and just have users install it from elsewhere.  If that isn't possible, fzi_icl_can should probably try to detect whether libpcan is already installed, and if so, not build its own version of it.

**Comment by fmauch** [comment]
Hey there,

The reason why we suggest to install an own version of libpcan is that by default, pcan is build with NETDEV support, however, we need the chardev version to work right now. As far as I know, Peak changed the default behaviour again in recent versions, but I don't know about the ROS packages.

However, as far as I remember, libpcan should not be installed from that package, but we offered a shell script to install it. I'll have to check that more thoroughly, then I'll come back to this issue.

**Comment by clalancette** [comment]
@fmauch I'm honestly not sure of the relationship between these things, only that they cause a conflict when trying to install both.  Thanks for taking a look, and let me know if I can help at all.

**Comment by fmauch** [comment]
Hi there, I had a deeper look into the problem today and hopefully have a fix on the way to use the libpcan ros package.

**Comment by clalancette** [comment]
Excellent!  Thanks a lot.

**Comment by fmauch** [comment]
I released a new package version (1.0.11) that uses the ros-kinetic-pcan package. I just tried it with a fresh live-system installation and it worked flawlessly, thus I'll mark this resolved. You'll just have to wait until the packages are synced again or use the shadow-fix repo.

**Comment by clalancette** [comment]
Great, thanks again!
